src/WLGDDetectorConstruction.cc

```diff
@@ -342,15 +363,26 @@ G4VPhysicalVolume* WLGDDetectorConstruction::SetupBaseline()
     G4Material* worldMaterial =
         G4NistManager::Instance()->FindOrBuildMaterial("G4_Galactic");
     G4Material* larMat = G4NistManager::Instance()->FindOrBuildMaterial("G4_lAr");
-    G4Material* concreteWallMat =
-        G4NistManager::Instance()->FindOrBuildMaterial("G4_CONCRETE");
     G4Material* airMat   = G4NistManager::Instance()->FindOrBuildMaterial("G4_AIR");
     G4Material* waterMat = G4NistManager::Instance()->FindOrBuildMaterial("G4_WATER");
     G4Material* steelMat =
         G4NistManager::Instance()->FindOrBuildMaterial("G4_STAINLESS-STEEL");
     G4Material* copperMat = G4NistManager::Instance()->FindOrBuildMaterial("G4_Cu");
 
-    // enriched Germanium from isotopes
+    auto C     = new G4Element("Carbon", "C", 6., 12.011 * g / mole);
+    auto O     = new G4Element("Oxygen", "O", 8., 16.00 * g / mole);
+    auto Ca    = new G4Element("Calcium", "Ca", 20., 40.08 * g / mole);
+    auto Mg    = new G4Element("Magnesium", "Mg", 12., 24.31 * g / mole);
+    
+    // Standard Rock definition, similar to Gran Sasso rock
+    // with density from PDG report
+    auto stdRock = new G4Material("StdRock", 2.65*g/cm3, 4);
+    stdRock->AddElement(O,  52.0*perCent);
+    stdRock->AddElement(Ca, 27.0*perCent);
+    stdRock->AddElement(C,  12.0*perCent);
+    stdRock->AddElement(Mg,  9.0*perCent);
+
+   // enriched Germanium from isotopes
```

Comment:
Think there are two copies of this code - first appears at line 147?

Reply:
Yes, there are. Not ideal but was easier to create the two required geometries self-contained. That can be factorised. Didn't get to that yet. I was mostly looking at production cuts after this PR. Also the primary event location looks strange according to the outputs. 
To be specific for this PR: we need a code that can actually run in order to check the tracking verbose 1 output. Thankfully, this does for me on the SCRTP.